cristin-person/src/main/java/no/unit/nva/cristin/person/query/organization/ListCristinOrganizationPersonsHandler.java

```diff
@@ -73,40 +77,20 @@ public ListCristinOrganizationPersonsHandler(CristinOrganizationPersonsClient ap
     
     @Override
     protected SearchResponse<Person> processInput(Void input, RequestInfo requestInfo, Context context)
-            throws ApiGatewayException {
+        throws ApiGatewayException {
 
         validateHasIdentifierPathParameter(requestInfo);
         validateQueryParameterKeys(requestInfo);
-
-        var identifier = getValidId(requestInfo);
-        var page = getValidPage(requestInfo);
-        var numberOfResults = getValidNumberOfResults(requestInfo);
-        var name =
-            getNameIfPresent(requestInfo).orElse(null);
-        var sort =
-            getSortIfPresent(requestInfo).orElse(null);
-
-        var requestQueryParams =
-            buildParamMap(identifier, page, numberOfResults, name, sort);
-
-        return attempt(() -> getAuthorizedSearchResponse(requestInfo, requestQueryParams))
```

Comment:
If getAuthorizedSearchResponse throws something like ApiGatewayException, could that be swallowed by the attempt and then fallback to open data? Maybe that was an issue even if we have retries in our Http Client fetch?